### ISSUE TITLE
iOS unable to archive [RN 0.63.3] - ".../AccessibilityResources.bundle" not found

### ISSUE THREAD

**Issue by cedced19** [facebook/react-native#30335]
Hi, I'm facing [the same kind of issue](https://github.com/facebook/react-native/issues/29709) since some days. The workarounds didn't work for me. Someone had the [same issue](https://github.com/shahen94/react-native-video-processing/issues/319) but didn't find any solution. Thank you for your help.

## Description

I am unable to archive the project for release: 
```
error: Resource "/Users/cedricjung/Library/Developer/Xcode/DerivedData/decouverto-asvavlxsbksyvhgovxmjpnlracmz/Build/Intermediates.noindex/ArchiveIntermediates/decouverto/BuildProductsPath/Release-iphoneos/React-Core/AccessibilityResources.bundle" not found. Run 'pod install' to update the copy resources script.
```

## React Native version:
```
System:
    OS: macOS 10.15.7
    CPU: (4) x64 Intel(R) Core(TM) i3-8100B CPU @ 3.60GHz
    Memory: 75.77 MB / 8.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.19.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.10.0 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.1, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
    Xcode: 12.1/12A7403 - /usr/bin/xcodebuild
  Languages:
    Java: Not Found
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.3 => 0.63.3 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```
## Steps To Reproduce

Archive [this](https://github.com/decouverto/les-balades-de-decouverto-ios) RN project.

## Link to my repository:

You can check all source code [there](https://github.com/decouverto/les-balades-de-decouverto-ios).

**Comment by cedced19** [comment]
Still exists when downgrading to react-native 0.63.2

**Comment by chgsilva** [comment]
having the same issue

**Comment by julianxlozano** [comment]
Has anyone figured this out? Stuck for weeks now.

**Comment by sdb1228** [comment]
The way I overcame this @julianxlozano was navigating to the actual script located at `vim <Project>/ios/Pods/Target\ Support\ Files/Pods-<Project>/Pods-<project>-resources.sh` and removing the require of the accessibility pod.  Was able to get an archive built (currently testing to see what side effects if any).  I figured removing the validation straight in the script was okay given the other thread actually says remove the pod in general.

**Comment by davidking1234** [comment]
Having the same issue. Is there any solution?

**Comment by error-try-again** [comment]
Had this issue several days ago, have no idea why it's happening. A bit bruteforce but I managed to find a workaround. Simply delete the duplicate React-Core-AccessibilityResources pod folder so you're left with the React-Core-603* version the clean/build, archive.

<img width="537" alt="error" src="https://user-images.githubusercontent.com/19685177/222868526-ead5e2a8-89e8-4274-8767-a6b6313f7978.png">

I'm still trying to find a workaround for my xcode cloud deployment. 

> Hi, I'm facing [the same kind of issue](https://github.com/facebook/react-native/issues/29709) since some days. The workarounds didn't work for me. Someone had the [same issue](https://github.com/shahen94/react-native-video-processing/issues/319) but didn't find any solution. Thank you for your help.
> ## Description
> 
> I am unable to archive the project for release:
> 
> ```
> error: Resource "/Users/cedricjung/Library/Developer/Xcode/DerivedData/decouverto-asvavlxsbksyvhgovxmjpnlracmz/Build/Intermediates.noindex/ArchiveIntermediates/decouverto/BuildProductsPath/Release-iphoneos/React-Core/AccessibilityResources.bundle" not found. Run 'pod install' to update the copy resources script.
> ```
> 
> ## React Native version:
> 
> ```
> System:
>     OS: macOS 10.15.7
>     CPU: (4) x64 Intel(R) Core(TM) i3-8100B CPU @ 3.60GHz
>     Memory: 75.77 MB / 8.00 GB
>     Shell: 5.7.1 - /bin/zsh
>   Binaries:
>     Node: 12.19.0 - /usr/local/bin/node
>     Yarn: Not Found
>     npm: 6.14.8 - /usr/local/bin/npm
>     Watchman: 4.9.0 - /usr/local/bin/watchman
>   Managers:
>     CocoaPods: 1.10.0 - /usr/local/bin/pod
>   SDKs:
>     iOS SDK:
>       Platforms: iOS 14.1, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
>     Android SDK: Not Found
>   IDEs:
>     Android Studio: Not Found
>     Xcode: 12.1/12A7403 - /usr/bin/xcodebuild
>   Languages:
>     Java: Not Found
>     Python: 2.7.16 - /usr/bin/python
>   npmPackages:
>     @react-native-community/cli: Not Found
>     react: 16.13.1 => 16.13.1 
>     react-native: 0.63.3 => 0.63.3 
>     react-native-macos: Not Found
>   npmGlobalPackages:
>     *react-native*: Not Found
> ```
> 
> ## Steps To Reproduce
> 
> Archive [this](https://github.com/decouverto/les-balades-de-decouverto-ios) RN project.
> ## Link to my repository:
> 
> You can check all source code [there](https://github.com/decouverto/les-balades-de-decouverto-ios).

**Comment by shahzore-qureshi** [comment]
THANK YOU @sdb1228 !!! THAT WORKED!!! You are a life saver!
